<!-- Standalone launch file for aruco marker recognition, starting asr_object_database. @author Pascal Meissner, Tobias Allgeyer. @version See SVN -->

<launch>

  <include file="$(find asr_aruco_marker_recognition)/launch/aruco_marker_recognition_core.launch" /> 
  
  <node name="asr_object_database" type="asr_object_database" pkg="asr_object_database" output="screen" />

</launch>
